sourcegraph/learn
# Removing forestry CMS

## Type of change
Tech debt

### What should this PR do?
Remove the Forestry directory

### Why are we making this change?
We are not currently using a CMS and don't currently have plans to use one. The directory on the repo can be confusing to devs who are onboarding. 

See also this [internal document](https://docs.google.com/document/d/1vyow8XQtk0XMqmRv7oXP-Lc16-Qcb2-RvHZtQ1ffiR0/edit#heading=h.ouou9cfmaxw) with discussion regarding the CMS.

### What are the acceptance criteria? 
Ensure that site still works as expected.

### How should this PR be tested?
Test the site locally to ensure that everything works as expected, or review the deploy preview to ensure that everything works as expected.

## Pull request process

**Reviewers**:

1. Test functionality using the criteria above.
2. Offer tips for efficiency, feedback on best practices, and possible alternative approaches and things that may not have been considered.
3. For shorter, "quick" PRs, use your best judgement on #​2.
4. Use a collaborative approach and provide resources and/or context where appropriate.
5. Provide screenshots/grabs where appropriate to show findings during review.

**Reviewees**:

1. Prefer incremental and appropriately-scoped changes.
2. Leave a comment on things you want explicit feedback on.
3. Respond clearly to comments and questions.
